coz-shop/src/components/Product.js

```diff
@@ -1,8 +1,22 @@
+import { useState } from "react"
 
-const Product = ({obj}) =>{
+const Product = ({obj,bookmarkHandle,bookmark}) =>{
+    const [modalOn,setModalOn] = useState(false)
+    const modalHandle = () =>{
+       setModalOn(!modalOn)
+    }
     return <article>
-        <img src={obj.image_url ? obj.image_url : obj.brand_image_url? obj.brand_image_url: ""} alt={obj.title} className="product_img"></img>
-        <img src={process.env.PUBLIC_URL + '/bookmark_ico-off.png'} className="on-off"/>
+        {modalOn? <section className="modal_Container">
+         <img src={obj.image_url ? obj.image_url : obj.brand_image_url? obj.brand_image_url: ""} alt={obj.title} className="modal_img"/>
+         <span className="closeBtn" onClick={modalHandle}><img src={process.env.PUBLIC_URL + '/Vector.png'}></img></span> <span className='modal_font'>{obj.type === 'Category'? '# '+obj.title : obj.title ? obj.title : obj.brand_name ? obj.brand_name : ''}</span>
```

Comment:
```suggestion
         <span className="closeBtn" onClick={modalHandle}><img src={process.env.PUBLIC_URL + '/Vector.png'}></img></span> <span className='modal_font'>{obj.type === 'Category'? '# '+obj.title : obj.title ? obj.title : obj.brand_name ? obj.brand_name : ''}</span>
```

- 여기 lint도 가능하면 해주면 좋을 것 같습니다.
- 삼항연산자가 3,4회 연속으로 쓰여진 부분이 있는데, 가능하면 유틸 함수 하나 만들어서 정리해주시면 좋겠습니다.